customisation/alignment.md

```diff
@@ -1,6 +1,27 @@
 ---
 parent: Customisation
-layout: coming_soon
 title: Alignment
-description:
+description: Linking corresponding sentences in the input and output languages
 ---
+
+**Alignment** is the process of identifying and linking the corresponding sentences in the input and output languages.
+
+Alignment can be used to create [parallel data](/parallel-data).
+The aligned parallel corpora are then used to train machine translation models.
+The goal is to improve machine translation accuracy through pattern and regularity recognition in data.
```

Comment:
Maybe to make it simpler:

> The goal is to improve machine translation accuracy by recognizing patterns and their frequency in data.

It may be a silly update, but the term "regularity", although accurate, made me think of academic/research speech.